src/MongoDB.Driver.Core/Core/Operations/ChangeStreamOperation.cs

```diff
@@ -62,34 +78,67 @@ public class ChangeStreamOperation<TResult> : IChangeStreamOperation<TResult>
         private int? _batchSize;
         private Collation _collation;
         private readonly CollectionNamespace _collectionNamespace;
+        private readonly DatabaseNamespace _databaseNamespace;
```

Comment:
The change stream operation targets either a client, a database or a collection.

When targetting a client both the collection and database namespaces will be null.